clarity_ext/context.py

```diff
@@ -124,6 +131,17 @@ def output_result_file_by_id(self, file_id):
     def output_result_files(self):
         return self.artifact_service.all_output_files()
 
+    @property
+    def user_initials(self):
+        return self.current_user.initials
+
+    @property
+    def pid(self):
+        return self.session.current_step_id
+
+    def today(self, format):
```

Comment:
I wonder the same here as above. I don't think we need to hide too much of core Python stuff for the user. The main idea with the `context` is I believe to hide boilerplate related to Clarity. I think we will otherwise go down the rabbit hole of proxying everything Python has to offer.